src/frontend/sider/Fagsak/Fagsaklinje/Behandlingsmeny/OpprettBehandling/useOpprettBehandling.ts

```diff
@@ -141,7 +143,9 @@ const useOpprettBehandling = ({
             },
             response => {
                 if (response.status === RessursStatus.SUKSESS) {
-                    oppdaterKlagebehandlingerPåFagsak();
+                    queryClient.invalidateQueries({
+                        queryKey: HentMinimalFagsakQueryKeyFactory.fagsak(fagsakId),
+                    });
```

Comment:
Her skal du kanskje bruke `HentKlagebehandlingerQueryKeyFactory`

```suggestion
                    queryClient.invalidateQueries({
                        queryKey: HentKlagebehandlingerQueryKeyFactory.fagsak(fagsakId),
                    });
```

Reply:
Herfra så kan fagsakId være undefined, så da blir HentKlagebehandlingerQueryKeyFactory til å ta inn fagsakId: number | undefined.